xml/depl_maintenance.xml

```diff
@@ -312,6 +312,175 @@
   </orderedlist>
  </sect1>
 
+ <sect1 xml:id="sec.depl.maintenance.recover.compute.node.failure">
+  <title>Recovering from &compnode; Failure</title>
+  <para>
+   The following procedure assumes that there is at least one &compnode;
+   already running. Otherwise, see <xref
+   linkend="sec.depl.maintenance.bootstrap.compute.plane" />.
+  </para>
+  <procedure>
+   <step>
+    <para>
+     If the &compnode; failed, it should have been fenced. Verify that this is
+     the case. Otherwise, check <filename>/var/log/pacemaker.log</filename> on
+     the Designated Co-ordinator to determine why the &compnode; was not fenced. The
```

Comment:
Our SLEHA docs use "Designated Coordinator" (no hyphen) which also makes more sense to me.